docker/docker-compose.integration-timescale.yml

```diff
@@ -0,0 +1,12 @@
+services:
+  timescale_example:
+    image: timescale/timescaledb:latest-pg12
```

Comment:
Good question, I personally viewed this as a starting point that we could continue to refine based on customer needs.  I used `12` because this is based on our Postgres connector which also uses `12`. There very well may be 14 functionality we need to add in the future.  As an aside, there's a lot of planned functionality that the postgres connector itself doesn't yet support. 

I did just attempt the 14 quickly and am running into `"SCRAM authentication requires libpq version 10 or above" ` issues. I haven't tried to troubleshoot this further.